book/B-embedding-git/sections/dulwich.asc

```diff
@@ -40,5 +40,4 @@ porcelain.log('.', max_entries=1)
 
 ==== Further Reading
 
- * The official API documentation is available at https://www.dulwich.io/docs/api/[].
- * Official tutorial at https://www.dulwich.io/docs/tutorial[] has many examples of how to do specific tasks with Dulwich.
+The API documentation, tutorial, and many examples of how to do specific tasks with Dulwich available on the official website https://www.dulwich.io[].
```

Comment:
```suggestion
The API documentation, tutorial, and many examples of how to do specific tasks with Dulwich are available on the official website https://www.dulwich.io[].
```

I think it reads better with the article added.

Reply:
Thanks! Fixed.